app/form_models/publishers/job_listing/applying_for_the_job_form.rb

```diff
@@ -14,6 +14,11 @@ class Publishers::JobListing::ApplyingForTheJobForm < Publishers::JobListing::Va
 
   private
 
+  def override_enable_job_applications
+    override_enable_job_applications_for_local_authority!
+    override_enable_job_applications_when_vacancy_published! unless params[:current_organisation].local_authority?
```

Comment:
Do we need this `unless`?

Reply:
Well I just figured it saved executing (is that the correct term?)  `override_enable_job_applications_when_vacancy_published!` if the user is signed in as an LA, as enable_job_applications would've already been overridden